packages/quick_actions/quick_actions_android/example/android/app/src/androidTest/java/io/flutter/plugins/quickactionsexample/QuickActionsTest.java

```diff
@@ -1,23 +1,169 @@
 // Copyright 2013 The Flutter Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
 
 package io.flutter.plugins.quickactionsexample;
 
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```

Comment:
This violates style: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
Done.